### ISSUE TITLE
ppp

### ISSUE THREAD

**Issue by mr-omi** [Nishant11769/team_project_planner_tool#1]
Hello, Nishant I got an error while executing your project on Postman. I have created and updated users everything works fine, now I am trying to create Team using (POST) "http://127.0.0.1:8000/teams/create/" and entered Input as you explained in readme file. but I am getting above error. Also I am using Sqlite3 but I don't think that's the reason of error.  I looked all over internet couldn't find the solution. Can you able to solve this error ?

Basically Error is Saying that its unable to convert entered users  (List) into an models users (array)
